RecoTracker/MeasurementDet/plugins/MeasurementTrackerEventProducer.cc

```diff
@@ -105,6 +120,47 @@ MeasurementTrackerEventProducer::updatePixels( const edm::Event& event, PxMeasur
     }
   }
 
+  if (!theBadPixelFEDChannelsLabels.empty()) {
+    edm::Handle<PixelFEDChannelCollection> pixelFEDChannelCollectionHandle;
+    for (const edm::EDGetTokenT<PixelFEDChannelCollection>& tk: theBadPixelFEDChannelsLabels) {
+      if (!event.getByToken(tk, pixelFEDChannelCollectionHandle)) continue;
+      int i=0;
+      for (const auto& disabledChannels: *pixelFEDChannelCollectionHandle) {	
+	PxMeasurementDetSet::BadFEDChannelPositions positions;
+	for(const auto& ch: disabledChannels) {
+	  const sipixelobjects::PixelROC *roc_first=NULL, *roc_last=NULL;
+	  sipixelobjects::CablingPathToDetUnit path = {ch.fed, ch.link, 0};
+	  for (path.roc=1; path.roc<=(ch.roc_last-ch.roc_first)+1; path.roc++) {
+	    const sipixelobjects::PixelROC *roc = cablingMap.findItem(path);
```

Comment:
I'm a bit surprised that the loop is from 1 to (last-first)+1 instead of from first to last or something, but on the other hand I don't know the details of the cabling map.